packages/shared/src/hooks/squads/useSquadPendingPosts.ts

```diff
@@ -47,7 +47,7 @@ export const useSquadPendingPosts = ({
     queryKey: generateQueryKey(
       RequestKey.SquadPostRequests,
       user,
-      squadId ?? 'all',
+      squadId,
```

Comment:
I'd rather have this kept, makes it easier to understand 🙏 
```suggestion
      squadId ?? 'all',
```